README.md

```diff
@@ -121,6 +121,13 @@ vault write auth/kubernetes/role/vault-secrets-operator \
   ttl=24h
 ```
 
+In the `values.yaml` add vault.authMethod=kubernetes. To configur the vault-secrets-operator to use the service account to authenticate instead of the default token method.
```

Comment:
Thanks for your contribution to the Vault Secrets Operator. Can you change the wording a little bit please:

```suggestion
When you deploy the Vault Secrets Operator via Helm chart you have to set the `vault.authMethod` property to `kubernetes` in the `values.yaml` file, to use the Kubernetes auth method instead of the default Token auth methods.
```

Reply:
Yes done!

Reply:
Thank you 🙂